### ISSUE TITLE
[FEAT]: Add shuffle to "play mode"

### ISSUE THREAD

**Issue by wrybread** [nova-video-player/aos-AVP#865]
### Description

It would be nice to have a shuffle mode, and I think an easy way to add it would be under "play mode". The current options are "single", "folder", "repeat single", "repeat folder" and "binge watching". Would be nice to have options for "folder shuffled" and "repeat folder shuffled" that would play all the files in random order.

Seems like a nice simple and clean way to add shuffle to the interface.

**Comment by Pentaphon** [comment]
Who shuffles movies? Who shuffles TV episodes? I don't see the use for this outside of music.

**Comment by wrybread** [comment]
A common use case is playing a folder of music videos. Or queing a random episode of a show. Or in my immediate use case playing a folder of videos shot by a drone which I'm trying to play randomly as a sort of screensaver.

Nova isn't just for playing movies or TV episodes, it's a general video player, and the ability to shuffle is a very common feature among video players. I can't think of a single other video player that doesn't have the ability. It's about the only glaring omission to this great program.

**Comment by Pentaphon** [comment]
There's a new UI in the works so hey, perhaps shuffle will be added to that but I'm not sure it's a priority.

**Comment by wrybread** [comment]
Fair enough about it not being a priority of course. But one note, I was thinking about what you said "Who shuffles movies? Who shuffles TV episodes? I don't see the use for this outside of music.". By that same token who repeats a movie? Who repeats a folder of shows? And yet the ability to do so is already in Nova. Not talking to you specifically here, but it seems to me that the ability to repeat a single video or folder of videos but not the ability to shuffle points to an inconsistent or incomplete feature set. Easily remedied of course. 

And if anyone can think of another major video player that doesn't have a shuffle mode I'd be interested in hearing it.

**Comment by MichaelD4824** [comment]
> Who shuffles movies? Who shuffles TV episodes? I don't see the use for this outside of music.

With my son, we put his shows on shuffle on another player multiple times a day. There’s 300 episodes of Paw Patrol, we use shuffle so we don’t have to constantly find whatever episode he wants to watch. We use Nova on his kids tablet for the car, this would be an awesome feature to have.

**Comment by wrybread** [comment]
I shuffle movies too. The ability to watch something random is extremely
useful and available in every other movie player that I know of.

**Comment by Mightyconan** [comment]
I would also love to have shuffle mode added.  Currently on my Android TV the only (reliable) video player app that I can find is VLC which has a ridiculous database footprint of over 1GB (with thumbnails disabled!) causing my TV to run very slow.   Nova is in the region of 150MB, which is great.  But like someone else mentioned above, it's nice to have music videos on shuffle, and my wife also likes to put random TV series on a shuffle of episodes in the background while working or struggling to sleep (strange I know but it works for her).   If we have a movie night then a 'random' movie from our "unwatched" folder would be a fun thing to do too.
